DEVELOPMENT.md

```diff
@@ -202,3 +202,48 @@ Additional information which is really important to how the application runs can
 be found in the manifest:
 
 > `$ bundle exec 3scale_backend manifest`
+
+## Running with RubyMine
+
+Using RubyMine scripts can help you run both Worker and Listener.
+
+For that, you'll need to create the following configurations: 
+
+### Worker
+
+First you'll have to set a name. We recommend using `worker`.
+
+Then you'll have to add the following config:
+
+```
+Ruby script: /Users/lvillenj/Projects/apisonator/bin/3scale_backend_worker
+Script arguments: --debug
+Working directory: /Users/lvillenj/Projects/apisonator
+Environment variables: RACK_ENV=development
+```
+
+### Puma Listener
+
+First you'll have to set a name. We recommend using `listener (puma)`.
+
+Then you'll have to add the following config:
+
+```
+Ruby script: /Users/lvillenj/Projects/apisonator/bin/3scale_backend
+Script arguments: -s puma -X "-w 0" start -p 3001
+Working directory: /Users/lvillenj/Projects/apisonator
+Environment variables: RACK_ENV=development;LISTENER_WORKERS=1
+```
+
+### Falcon Listener
```

Comment:
Nice! What is this used for though? Maybe a line of explanation why and when we want to run each one of those.

Reply:
These configs are the ones you would use with RubyMine configurations and they just intend to make life easier for developers, for more details users can go to `README.md` I'd say, WDYT?

Reply:
Ok, maybe actually link from README to DEVELOPMENT about these. The README instructions about `make` seem to be broken though. I hope the `docker` instructions aren't.

Reply:
@akostadinov The listener runs Puma on sync mode and Falcon on async mode.